static/app/views/insights/browser/webVitals/types.tsx

```diff
@@ -137,3 +138,36 @@ export const DEFAULT_INDEXED_SPANS_SORT: Sort = {
   kind: 'desc',
   field: 'timestamp',
 };
+
+export const WEB_VITAL_PERFORMANCE_ISSUES: Record<
+  WebVitals,
+  Array<keyof typeof ISSUE_TYPE_TO_ISSUE_TITLE>
+> = {
+  lcp: [
+    'performance_render_blocking_asset_span',
+    'performance_uncompressed_assets',
+    'performance_http_overhead',
+    'performance_consecutive_http',
+    'performance_n_plus_one_api_calls',
+    'performance_large_http_payload',
+    'performance_p95_endpoint_regression',
+  ],
+  fcp: [
+    'performance_render_blocking_asset_span',
+    'performance_uncompressed_assets',
+    'performance_http_overhead',
+    'performance_consecutive_http',
+    'performance_n_plus_one_api_calls',
+    'performance_large_http_payload',
+    'performance_p95_endpoint_regression',
+  ],
+  inp: [
+    'performance_http_overhead',
+    'performance_consecutive_http',
+    'performance_n_plus_one_api_calls',
+    'performance_large_http_payload',
+    'performance_p95_endpoint_regression',
+  ],
+  cls: [],
+  ttfb: ['performance_http_overhead'],
+} as const;
```

Comment:
This makes sense, but I bet people won't remember to update this. I think a more flexible solution here would be to:

1. Add a field to the `GroupType` class in Python that specifies whether a given group type affects page load performance or not
2. Add a filter to issue search that allows to search for issues that affect page load

I think that approach is better because when we add more issue detectors (apparently this is planned) and when users start creating their own issue detectors based on metrics we can make sure that _those_ issues also show up in these dropdowns somehow

That's a lot more work, obviously, but worth considering, IMO!

Reply:
Hmm, interesting solution. I can look into this! Would you mind if I explored this separately and handle this in separate prs? Should be simple to pivot the frontend once I've dug into this a bit more